python_on_whales/components/buildx/cli_wrapper.py

```diff
@@ -208,7 +208,7 @@ def build(
         allow: List[str] = [],
         attest: Optional[Dict[str, str]] = None,
         build_args: Dict[str, str] = {},
-        # TODO: build_context
+        build_contexts: Dict[str, str] = {},
```

Comment:
```suggestion
        build_contexts: Dict[str, Union[str, ValidPath]] = {},
```
Any path-like object will also be accepted for local directories